src/android/aidl/example-service/interface.md

```diff
@@ -0,0 +1,28 @@
+# AIDL Interfaces
+
+You declare the API of your service using an AIDL interface:
+
+_birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl_:
+
+```java
+{{#include ../birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl:IBirthdayService}}
+}
+```
+
+_birthday_service/aidl/Android.bp_:
+
+```javascript
+{{#include ../birthday_service/aidl/Android.bp}}
+```
+
+Add `vendor_available: true` if your AIDL file is used by a binary in the vendor
```

Comment:
Or `vendor: true` is it's _only_ for vendor, but the audience might already know that.

Reply:
I'm not sure we even need this comment about `vendor_available` in this slide to begin with since it's more of a general AOSP thing and isn't really relevant to anything else we talk about. I'm removing it.